guix/README.md

```diff
@@ -0,0 +1,19 @@
+# Reproducible Notebook with Guix
```

Comment:
Please make this section better.

Reply:
Sorry I figured you'd just be [reading on github](https://github.com/zkSNACKs/WabiSabi/blob/amount-org-notebook/Amount%20Org.ipynb) to start so I didn't take much care.

I expanded the instructions a bit, I hope it's enough but i'm not sure if this will work under WSL and I've got no windows machine to try.

If you are interested in trying out guix (highly reccomended IMO) I'm happy to help get started with it but it shouldn't really be a requirement, it's just my personal preference and happens to support reproducibility well so I like using it for dev environments